dask_cuda/local_cuda_cluster.py

```diff
@@ -54,13 +54,13 @@ def __init__(
         local_directory=None,
         **kwargs,
     ):
-        if n_workers is None:
-            n_workers = get_n_gpus()
         if CUDA_VISIBLE_DEVICES is None:
             CUDA_VISIBLE_DEVICES = cuda_visible_devices(0)
         if isinstance(CUDA_VISIBLE_DEVICES, str):
             CUDA_VISIBLE_DEVICES = CUDA_VISIBLE_DEVICES.split(",")
         CUDA_VISIBLE_DEVICES = list(map(int, CUDA_VISIBLE_DEVICES))
+        if n_workers is None:
+            n_workers = len(CUDA_VISIBLE_DEVICES)
```

Comment:
What if the user doesn't pass `CUDA_VISIBLE_DEVICES`?

Reply:
I believe that it is handled with the set of checks above just above.

Reply:
You're right. The `cuda_visible_devices(0)` got me confused, sorry for the false alarm.